### DISCUSSION TITLE
No idea of where to get the "Mii QR Key"

### DISCUSSION THREAD

**Discussion by Mr-Socket-bit** [Genwald/MiiPort#6] [Questions] [answered]
Google isn't very helpful. The closest thing i found was about some AES key for the Citra emulator, but that doesn't seem to have worked either.

**Comment by Genwald** [comment, marked as answer]
Sharing private encryption keys like this can be a bit of a weird thing legally, which is why I make the user provide them instead of including them in this program. Because of that I want to keep discussion here of where to get keys like this somewhat vague as well.

It would be the key that Citra calls `slot0x31KeyN`. If you need any extra help with this feel free to DM me on discord, I am `Genwald#1744`.
